### ISSUE TITLE
Communicate "beakerbrowser" on the local network

### ISSUE THREAD

**Issue by wuniversales** [beakerbrowser/beaker#1513]
I do not know if this function is currently performed, just in case I comment.

It would be nice to be able to communicate "beakerbrowser" with other browsers "beakerbrowser" of the local network, and if the requested website is available locally, obtain the website locally without going to the internet.

**Comment by pfrazee** [comment]
Yeah this is something I want to get into at some point, havent had a chance to hit it yet.
